yew-macro/tests/macro/html-tag-fail.rs

```diff
@@ -56,7 +56,6 @@ fn compile_fail() {
     html! { <a href?="href" /> };
     html! { <a href?=Some(NotToString) /> };
     html! { <input checked?=Some(false) /> };
-    html! { <input class?=() /> };
```

Comment:
It did become possible now! Maybe there is something to do that would make it magically forget empty `Classes`

Reply:
This was deliberately made impossible because `Option<T>` was already supported for `Classes` and it would completely break the syntax.
With these changes it makes sense for it to be possible now :+1: